debian/marquee/nginx/conf/whatwg.conf

```diff
@@ -0,0 +1,42 @@
+listen 443 ssl http2;
+listen [::]:443 ssl http2;
+
+ssl_session_cache shared:SSL:10m;
+ssl_session_timeout 5m;
+
+ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+
+ssl_prefer_server_ciphers on;
+
+ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS';
+
+ssl_dhparam /etc/ssl/certs/dhparam.pem;
+
+gzip on;
+gzip_disable "msie6";
+
+gzip_vary on;
+gzip_proxied any;
+gzip_comp_level 6;
+gzip_buffers 16 8k;
+gzip_http_version 1.1;
+gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml image/svg+xml application/xml+rss text/javascript;
+
+include whatwg-headers.conf;
+
+location ~* \.(?:ico|css|js|gif|jpe?g|png|svg)$ {
+    expires max;
+    add_header Pragma public;
+    include whatwg-headers.conf;
+}
+
+location /commit-snapshots {
+    autoindex on;
+    autoindex_localtime off;
+    autoindex_exact_size off;
+}
+
+include mime.types;
```

Comment:
Can you add a comment about why this is necessary given that it's also included in the main nginx.conf?